documentation/library_guide/kernel_templates/kernel_configuration.rst

```diff
@@ -0,0 +1,39 @@
+Kernel Configuration
+####################
+
```

Comment:
It would be nice to have an intro sentence before the code snippet.

Reply:
I followed a common structure used in cppreference (see more details here https://github.com/oneapi-src/oneDPL/pull/1328#discussion_r1451025869). 

Should I still add an intro sentence?

Reply:
Yes, see the comment in radix_sort.

Reply:
Done